admin/tabs/activity/clienterrors/ClientErrorModel.js

```diff
@@ -22,16 +25,56 @@ export class ClientErrorModel {
 
     @observable.ref detailRecord = null;
 
+    // TODO: Create pref
```

Comment:
Not sure. If so need a pref in the bootstrap

Reply:
Skip the pref for now.  We are enhancing our API for prefs like this anyway, so shortly it could be easy to just throw it in localStorage, which makes it even lower cost.

Reply:
Great thanks